sql/src/test/java/org/apache/druid/sql/calcite/CalciteQueryTest.java

```diff
@@ -602,6 +602,35 @@ public void testSafeDivide()
     );
   }
 
+  @Test
+  public void testDiv()
+  {
+    cannotVectorize();
+    final Map<String, Object> context = new HashMap<>(QUERY_CONTEXT_DEFAULT);
+
+    testQuery(
+        "select m1, div(m1, 2) from foo",
+        context,
+        ImmutableList.of(
+            newScanQueryBuilder()
+                .dataSource(CalciteTests.DATASOURCE1)
+                .intervals(querySegmentSpec(Filtration.eternity()))
+                .virtualColumns(expressionVirtualColumn("v0", "div(\"m1\",2)", ColumnType.LONG))
```

Comment:
Can we add one more test with `div(expression,expression)` just for sakr of sanity. Rest LGTM

Reply:
Thanks for taking a look. I added one more doing `div(cnt+2, cnt+1)`.